config/locales/export/csv/project.en.yml

```diff
@@ -20,6 +20,10 @@ en:
           school_dfe_number: DfE number
           school_name: School name
           school_type: School type
+          school_phase: School phase
+          local_authority_name: Local authority name
+          region: Region
+          advisory_board_date: Advisory board date
```

Comment:
These keys are all duplicates - they already exist in the file, so we can remove them.